lib/dartservices_client/v1.dart

```diff
@@ -1,3 +1,5 @@
+// This is a generated file (see the discoveryapis_generator project).
```

Comment:
Ok, that's fine, but if I put the code out now, the PR is not testable anymore, shall I leave it in for now, so you can still test it ? Then I will rebase the PR, once it is landed at dart-services and here in the master.